scripts/seed_data.py

```
@@ -0,0 +1,95 @@
+from django.utils import timezone
+
+from accounts.models import User, Profile
+from classrooms.models import Classroom, ClassroomTeacher, Enrollment
+
+
+def run():
+    # Get the absolute path to the dummy_data.json file
+    base_dir = os.path.dirname(os.path.abspath(__file__))
+    file_path = os.path.join(base_dir, 'dummy_data.json')
+
+    with open(file_path, 'r') as file:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)